apps/web/src/server/lib/msal.js

```diff
@@ -33,7 +33,7 @@ export const msalClient = new msal.ConfidentialClientApplication({
 				}
 			},
 			piiLoggingEnabled: false,
-			logLevel: msal.LogLevel.Verbose
+			logLevel: msal.LogLevel.Warning
```

Comment:
totally unrelated to the task but since I started running authentication on my local env I noticed the huge amount of useless logs that MSAL shows. This should keep all environments cleaner